src/Model/ValueObject/Web/InvalidEmailAddress.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\ValueObject\Web;
+
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\ApiProblem\ConvertsToApiProblem;
+use Exception;
+
+final class InvalidEmailAddress extends Exception implements ConvertsToApiProblem
```

Comment:
Maybe it's better to derive from `InvalidArgumentException` to ensure that other catch statements that rely on `InvalidArgumentException` still work. Unless we can ensure that calling `EmailAddress` with a failing `FILTER_VALIDATE_EMAIL` has no unexpected side effects?